app/uk/gov/hmrc/cataloguefrontend/CatalogueController.scala

```diff
@@ -225,20 +228,24 @@ class CatalogueController @Inject()(
             userManagementConnector.getTeamDetails(teamName),
             leakDetectionService.repositoriesWithLeaks,
             serviceDependencyConnector.dependenciesForTeam(teamName),
-            serviceDependencyConnector.getCuratedSlugDependenciesForTeam(teamName, SlugInfoFlag.ForEnvironment(Environment.Production))).mapN {
-            (teamMembers, teamDetails, reposWithLeaks, masterTeamDependencies, prodDependencies) =>
+            serviceDependencyConnector.getCuratedSlugDependenciesForTeam(teamName, SlugInfoFlag.ForEnvironment(Environment.Production)),
+            if (hideArchivedRepositoriesFromTeam) teamsAndRepositoriesConnector.archivedRepositories else Future.successful(Nil)).mapN {
+            (teamMembers, teamDetails, reposWithLeaks, masterTeamDependencies, prodDependencies, archivedRepos) =>
+              val reposToHide: Seq[String] = archivedRepos.map(_.name)
```

Comment:
could move the `.map(_.name)` above so the data is ready for `.mapN` i.e `.mapN {(.., reposToHide) => }`

Reply:
Done.